$OpenBSD: patch-ghc_rts_obscure_ccall_s,v 1.3 2003/09/01 20:32:14 espie Exp $
The assembly code to go in .text, replacing the opcodes for this
code that were stored in an array in .data land, by a C hacker.
See Adjustor.c

--- /dev/null	Thu Jul 10 14:39:57 2003
+++ ghc/rts/obscure_ccall.s	Thu Jul 10 14:48:47 2003
@@ -0,0 +1,6 @@
+        .text
+        .globl __obscure_ccall_ret_code
+
+__obscure_ccall_ret_code:
+        addl $4, %esp
+        ret
